README.md

```diff
@@ -144,8 +144,8 @@ Example:
 
 👤 **Benson Kiai:**
 
-- GitHub: [@BenMKT](https://github.com/BenMKT/Portfolio)
-- linkedin: [@BenK](www.linkedin.com/in/bensonkiai)
+- GitHub: [@BenMKT](https://github.com/BenMKT/myPortfolio)
+- Linkedin: [@BenK](www.linkedin.com/in/bensonkiai)
```

Comment:
- [ ] Your LinkedIn path is actually following github path which makes it difficult to access your LinkedIn page.

This is what I see when I click your LinkedIn;
`https://github.com/BenMKT/myPortfolio/blob/desktop-version/www.linkedin.com/in/bensonkiai`

To fix this, I think you should attach `https` to your LinkedIn route like this:
`https://www.linkedin.com/in/bensonkiai`

Reply:
- Inserted `https` to my Linkedin path to work correctly.